MozillaFoundation/foundation.mozilla.org
# Remove `.localized` from `profile_page.html`

# Description



Remove `.localized` methods from `libraries/profile_page.html`.

Since the profiles are already localized using `utils.localize_queryset`, there is no need to use `.localized` at the template level.

## Testing

1. Translate a profile to French language
2. Navigate to "fr/research/authors"
3. Click on the translated profile
4. At the author detail page, you should see the translated tagline and introduction

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/86f7135b-c627-4a7c-ad37-2be4cbcceb91)


Related PRs/issues: #10017
